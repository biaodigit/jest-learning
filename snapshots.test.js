const generateConfig = () => {
    return {
        server: 'http://localhost',
        host: '8000',
        domain: 'localhost',
        time: '111223'
    }
}

test('generate config', () => {
    expect(generateConfig()).toMatchSnapshot()
})