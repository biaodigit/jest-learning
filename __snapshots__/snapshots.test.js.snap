// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate config 1`] = `
Object {
  "domain": "localhost",
  "host": "8000",
  "server": "http://localhost",
  "time": "111223",
}
`;
